ts/Extensions/Debugger/ErrorMessages.ts

```diff
@@ -108,6 +108,10 @@ const errorMessages: Record<string, Record<string, string>> = {
     "34": {
         "title": "Unknown time zone",
         "text": "<h1>Unknown time zone</h1><p>This error occurs if the browser doesn't recognize the <a href=\"https://api.highcharts.com/highcharts/time.timezone\">timezone</a> option. Possible workarounds are to use a time zone definition that all browsers recognize, or create your own <a href=\"https://api.highcharts.com/highcharts/time.getTimezoneOffset\">getTimezoneOffset</a> callback.</p>"
+    },
+    "35": {
+        "title": "Missing **highcharts.css** file",
+        "text": "<h1>Missing<b>highcharts.css</b> file</h1><p>This error occurs if you enable <a href=\"https://api.highcharts.com/highcharts/chart.styledMode\">styled mode</a> (<code>chart.styledMode = true</code>) without including the required **highcharts.css** file. Without this stylesheet, charts cannot be rendered correctly.</p><p>To fix this, make sure to load the official stylesheet in your page: <a href=\"https://www.highcharts.com/docs/chart-design-and-style/style-by-css\">Style by CSS</a>.</p>"
```

Comment:
Looks like the gulp task generating the messages is not handling bold text well - it trims the starting space causing text to merge like:

<img width="394" height="84" alt="Image" src="https://github.com/user-attachments/assets/52658c49-a2d2-45d1-a0e0-c6398313dc30" />